metals/src/main/scala/scala/meta/internal/pantsbuild/commands/SharedCommand.scala

```diff
@@ -89,27 +84,6 @@ object SharedCommand {
     }
   }
 
-  def symlinkProjectViewRoots(project: Project): Unit = {
```

Comment:
Can we still keep these symlinks? I'd like to get user feedback on whether they're useful or not

Reply:
Sure, this was just a proposal, It's up to you which option you prefer